### PR TITLE
execute commands in predefined list of projects

### PR DESCRIPTION
**Pull request by wingleung**

### CODE REVIEW

**Comment by addyosmani** [comment]
Hey! Thanks. Could you summarize the goals of your PR so it's clear to us what the intentions of changes are?

**Comment by anton-karlovskiy** [comment]
@addyosmani 
As you suggested earlier for multiple deployment, I think he applied iteration for build.sh and clean.sh, which are used to deploy the multiple demos with one command, so we can avoid duplication in build.sh and clean.sh scripts and it can also get it easy to update the scripts whenever we add a new demo to the project.

**Comment by wingleung** [comment]
Indeed, just removed the duplicate code and tried to list the projects in clearer way so adding a project to this flow is just adding 1 line in the `build.sh` en `clean.sh` files.

Another improvement could be to move the projects to a `projects` or `demos` folder. Shell scripts would just dynamically read all the folders in there and do the build or clean in each of them.

Depending on how far this project wants to go we might look into [lerna](https://github.com/lerna/lerna/) for this type of stuff. But in this stage, the shell scripts are more than sufficient and maybe even clearer to understand what's going on.

**Comment by addyosmani** [comment]
Thanks for the summary, @wingleung and @anton-karlovskiy! For now sticking with shell scripts and evaluating something more formal (Lerna) in the future sounds like a good plan. Appreciate the PR!
